projects/plugins/backup/jetpack-backup.php

```diff
@@ -5,7 +5,7 @@
  * Plugin URI: https://jetpack.com/jetpack-backup
  * Description: Easily restore or download a backup of your site from a specific moment in time.
  * Version: 1.8-alpha
- * Author: Automattic
+ * Author: Automattic - Backup team
```

Comment:
I think we've used "Jetpack" in there for the other plugins:
https://wordpress.org/plugins/jetpack-protect/
https://wordpress.org/plugins/jetpack-boost/

```suggestion
 * Author: Automattic - Jetpack Backup team
```

Or maybe

```suggestion
 * Author: Automattic - Jetpack Security team
```

Reply:
We do, yes; good catch. I think `Jetpack Backup team` is better for what we're going for here; we use Security for the Protect plugin.